Jint/Engine.cs

```diff
@@ -295,6 +295,17 @@ public void ResetStatementsCount()
             _statementsCount = 0;
         }
 
+        /// <summary>
+        /// Gets current statement count.
+        /// </summary>
+        public int StatementCount
```

Comment:
Should this be `ExecutedStatementCount`?

Reply:
Yours is a better name.